tests/data/table/tableUTest.cxxtest

```diff
@@ -468,4 +468,18 @@ public:
 
 		TS_ASSERT(ctable == c_ctable);
 	}
+
+	void test_get_input_col_data()
+	{
+		string input_file = tableUTest_dir + "dataset.ctable";
+		CTable ct = loadCTable(input_file);
+
+		vertex_seq first_col = ct.get_input_col_data(0);
+
+		vertex_seq expected;
+		expected.push_back(id::logical_true);
+		expected.push_back(id::logical_false);
```

Comment:
Note that C++11 allows you to write
```c++
vertex_seq expected{id::logical_true, id::logical_false};
```
instead of using `push_back`. What you wrote if obviously valid, and maybe you wanted the code to be shaped like a column, just saying in case you didn't know about that type of initialization.